aws-transfer-user/src/main/java/software/amazon/transfer/user/CreateHandler.java

```diff
@@ -0,0 +1,125 @@
+                                        progress,
+                                        request,
+                                        newModel,
+                                        proxy,
+                                        proxyClient,
+                                        callbackContext))
+                .then(
+                        progress ->
+                                new ReadHandler()
+                                        .handleRequest(
+                                                proxy,
+                                                request,
+                                                callbackContext,
+                                                proxyClient,
+                                                logger));
+    }
+
+    private ProgressEvent<ResourceModel, CallbackContext> createUser(
+            AmazonWebServicesClientProxy proxy,
+            ProxyClient<TransferClient> proxyClient,
+            String clientRequestToken,
+            ProgressEvent<ResourceModel, CallbackContext> progress) {
+        return proxy.initiate(
+                        "AWS-Transfer-User::Create",
+                        proxyClient,
+                        progress.getResourceModel(),
+                        progress.getCallbackContext())
+                .translateToServiceRequest(CreateHandler::translateToCreateRequest)
+                .makeServiceCall(this::createUser)
+                .handleError(
+                        (ignored, exception, client, model, context) ->
+                                handleError(CREATE, exception, model, context, clientRequestToken))
+                .progress();
+    }
+
+    private static CreateUserRequest translateToCreateRequest(final ResourceModel model) {
+        String homeDirectoryType =
+                Optional.ofNullable(model.getHomeDirectoryType())
+                        .orElse(HomeDirectoryType.PATH.name());
+
+        return CreateUserRequest.builder()
+                .serverId(model.getServerId())
+                .userName(model.getUserName())
+                .policy(model.getPolicy())
+                .role(model.getRole())
+                .homeDirectoryType(homeDirectoryType)
+                .homeDirectory(model.getHomeDirectory())
+                .homeDirectoryMappings(
+                        translateToSdkHomeDirectoryMappings(model.getHomeDirectoryMappings()))
+                .posixProfile(translateToSdkPosixProfile(model.getPosixProfile()))
+                .tags(translateToSdkTags(model.getTags()))
+                .build();
+    }
+
+    private CreateUserResponse createUser(
+            CreateUserRequest request, ProxyClient<TransferClient> client) {
+        try (TransferClient transferClient = client.client()) {
+            CreateUserResponse response =
+                    client.injectCredentialsAndInvokeV2(request, transferClient::createUser);
+            log("user created successfully", request.userName());
```

Comment:
Let's log the server ID as well (or the model's primaryIdentifier if that's available). Ditto for other user handler logs

Reply:
I can see if I should make the model available here to make it possible to log the Arn instead. I generate the Arn for a create user when the handler is called.